edge-hub/watchdog/rust.toolchain

```diff
@@ -0,0 +1 @@
+1.43.1
```

Comment:
it should be fine for watchdog to be on `stable` toolchain, b/c it does not use async. We use 1.43 for mqttd only to work around clippy issue with .await.